src/main/java/com/codeborne/selenide/proxy/SelenideProxyServer.java

```
@@ -169,6 +173,9 @@ public void addResponseFilter(String name, ResponseFilter responseFilter) {
   @Nullable
   @CanIgnoreReturnValue
   public ResponseFilter removeResponseFilter(String name) {
+    if (name.startsWith(SELENIDE_PROXY_FILER_PREFIX)) {
+      return responseFilters.get(name);
```

I would throw an exception instead of silently ignoring the request to remove.